build.gradle

```diff
@@ -98,6 +99,7 @@ private void configureAndroidLibrary(Project project) {
         compileSdkVersion rootProject.ext.compileSdkVersion
 
         defaultConfig {
+            multiDexEnabled true
```

Comment:
wooo

Reply:
Eh, ~sorta, haha :-D

Reply:
At one point this error got brought up early on in the discussion (when discussing extensively overloading our API methods for convenience), but we dismissed it because 60k is A LOT. At the time, we didn't think we would be adding a direct dependency on a third-party framework...

Reply:
Yea, we'd be busy coding to get to 65,536 ourselves. The vast majority of these come from our dependencies.

Reply:
Do you think this was due to the addition of the threeten library?  That would be unfortunate :/

Reply:
I think after we agreed to add rxJava as a core dependency, we already set a precedent. From that point on, going over the limit was kind of inevitable, so I wouldn't blame it on any particular library

Reply:
John had run this over the weekend - https://github.com/KeepSafe/dexcount-gradle-plugin

That probably added a few, but it definitely wasn't one of the big offenders. OkHttp and RxJava alone are almost half of it.